pkg/ui/src/Header/Header.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+	import Logo from './Logo.svelte';
+	export let title = 'Stock';
+	const links = [
+		{
+			label: 'Stock',
+			href: '/inventory/stock'
+		},
+		{
+			label: 'Inbound',
+			href: '/inventory/inbound'
+		},
+		{
+			label: 'Outbound',
+			href: '/inventory/outbound'
+		}
+	];
+	export let currentUrl = 'stock';
+</script>
+
+<div class="bg-gray-900 p-10 pt-5 ">
```

Comment:
Instead of using `pt` here, can't we have:
```html
<!-- Main container -->
<header class="w-full px-[70px] bg-gray-900">
  
  <!-- First row -->
  <div class="h-16 border-b border-sky-900 ...other classes">
    <!-- Logo and nav -->
  </div>

  <!-- Second row -->
  <div class="h-[168px] ...other classes">
    <!-- Title and action button -->
  </div>

</header>
```
This way the outermost container is in charge of setting the background, the full width and the padding on the sides.

The children: first and second row can stretch for the full width (as the padding is already set by the parent element and are in charge of setting their own height, aligning their children elements and first row renders the border bottom